ipatests/test_cmdline/test_help.py

```diff
@@ -73,6 +75,25 @@ def test_ipa_help():
     assert ctx.stderr == ''
 
 
+def test_ipa_help_without_cache():
+    """Test `ipa help` without schema cache. Skip if no cache dir"""
+    cache_dir = os.path.expanduser('~/.cache/ipa/schema/')
+    backup_dir = os.path.expanduser('~/.cache/ipa/schema.bak/')
+    shutil.rmtree(backup_dir, ignore_errors=True)
+    if os.path.isdir(cache_dir):
+        os.rename(cache_dir, backup_dir)
+    else:
+        pytest.skip('Cache dir {} is not present'.format(cache_dir))
```

Comment:
Why do you want to skip the entire test when the cache dir does not exist?

Reply:
Just this one test case,  however yes that is nonsense here. Just will need to avoid moving backup_dir to cache_dir when backup_dir does not exist.

Reply:
not to mention it is quite inconsistent with what I want to achieve here.